app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Listing Users</h1>
\ No newline at end of file
+<% @users.each do |user| %>
+    <div class="user">
+      <div class="user-photo">
+        <img src="<%= user.photo %>" alt="User Photo">
+      </div>
+      <div class="user-details">
+        <%= link_to user_url(user), class: 'name' do %>
+          <%= user.name %>
+        <% end %>
+        <p class="posts-no">
+          <%= "Number of posts: #{user.posts_counter}" %></p>
+      </div>
+    </div>
+  <% end %>
```

Comment:
- [ ] Kindly make sure the views of the app look like the provided wireframe

|Your website|Provided Wirframes|
|-|-|
|![Screenshot_20230629_003530](https://github.com/Odongo006/Rails_blog_App/assets/25227022/668ad737-9b38-46ca-86bd-0e5e9b371027)|![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png)|